src/progress/pct/compile.p

```diff
@@ -557,27 +569,27 @@ PROCEDURE displayCompileErrors PRIVATE:
   DEFINE VARIABLE c AS CHARACTER  NO-UNDO.
   DEFINE VARIABLE bit AS INTEGER NO-UNDO.
   DEFINE VARIABLE memvar AS MEMPTR NO-UNDO.
-  DEFINE VARIABLE include AS LOGICAL NO-UNDO.
+  DEFINE VARIABLE include AS LOGICAL NO-UNDO init true.
 
-  ASSIGN include = REPLACE(pcInit, CHR(92), '/') EQ REPLACE(pcFile, CHR(92), '/').
+  ASSIGN include = REPLACE(pcInit, CHR(92), '/') NE REPLACE(pcFile, CHR(92), '/').
 
   /* Checking if file is xcoded */
-  COPY-LOB FROM FILE (IF include THEN pcInit ELSE pcFile) FOR 1 TO memvar.
+  COPY-LOB FROM FILE pcFile FOR 1 TO memvar.
   bit = GET-BYTE (memvar, 1).
   SET-SIZE(memvar) = 0.
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Cannot reference "INITIAL" as "init" due to the "require-full-keywords" compiler option. (18494) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=rssw-oe%3Acompiler.warning.18494)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Abbreviated keyword init for INITIAL [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=rssw-oe-main%3Aeu.rssw.antlr.proparse.checks.AbbreviatedKeywords)